packages/react-server/src/metrics/test/metric.test.tsx

```diff
@@ -1,46 +1,13 @@
-import {Server} from 'http';
+import {saddle, unsaddle} from 'saddle-up/koa';
```

Comment:
Switching this test shows the comparison in niceness between `supertest` and `saddle-up`, we do way less rigamarole.